.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,264 @@
+name: PHPUnit
```

Comment:
The multi file is actually the new style and way to go.

Also brings in the nifty summary:
```
  summary:
    runs-on: ubuntu-latest
    needs: phpunit-sqlite

    if: always()

    name: phpunit-sqlite-summary

    steps:
      - name: Summary status
        run: if ${{ needs.phpunit-sqlite.result != 'success' }}; then exit 1; fi
```

Reply:
Damn, I added the single file in several repos. (but the other ones had non before so kinda okay?)

It seems your PRs only accounts for single file style so I thought it was correct.
Any change of adding the recommended setup to nextcloud/.github to be able to deploy updates for there? Or maybe we should look into reusable workflows to have this setup in only one place.